python_modules/dagster/dagster/__init__.py

```diff
@@ -670,6 +670,13 @@
 }
 
 
+# Use this to expose symbols from top-level dagster for testing purposes during development, before
+# we want to expose them to users.
+_HIDDEN: Final[Mapping[str, str]] = {
+    "Component": "dagster._components",
+}
```

Comment:
I think this is overkill. We don't need this

Reply:
We usually avoid exposing imports at the top-level public API until we're ready to release so they don't show up in typeahead-- that's not a good option here, you'd have to generate `from dagster._components import ...` in code location files. You want to just expose `Component` from top level?